doc/changes/changes-10.1.0.md

```diff
@@ -1,7 +1,8 @@
-# Virtual Schema Common Java 10.0.2, released 30.06.2020
+# Virtual Schema Common Java 10.1.0, released 02.07.2020
```

Comment:
Sebastian suggested to use the yyyy-mm-dd format for dates, does it apply here? Or was it for the dates in somewhere else?

Reply:
Yes, that is here.